Cargo.toml

```diff
@@ -134,6 +134,7 @@ quickcheck = "1.0"
 quickcheck_macros = "1.0"
 quote = "1.0"
 rand = "0.8"
+getrandom = "*" # use version requested by rand
```

Comment:
I think we can't publish with a star-dependency as well.

Reply:
Should I update it to the version that `rand` requests manually and add a comment about it?

Reply:
Yeah, I think this either needs to be a compatible range or minimal version.

Reply:
`rand` depends on `rand_core` and that depends on `getrandom@0.2`, so I set that in the `Cargo.toml` now

Reply:
Thanks!